packages/app/obojobo-document-engine/src/scripts/oboeditor/components/page-editor.js

```diff
@@ -46,10 +46,12 @@ class PageEditor extends React.Component {
 			showPlaceholders: true
 		}
 
+		this.imported = false
```

Comment:
Looking at the diff, this may not be needed any more?

Reply:
True